front/public/locales/fr/errors.json

```diff
@@ -195,6 +195,9 @@
     },
     "work_schedule": {
       "NameAlreadyUsed": "Un groupe de planches travaux avec le nom '{{name}}' existe déjà"
+    },
+    "app_health": {
+      "TimeoutError": "Le serveur n'a pas répondu à temps"
```

Comment:
I believe you'll have to change it also in [front/public/locales/en/errors.json](https://github.com/OpenRailAssociation/osrd/blob/88e6cd365311201b7b7468ef7b2eb9d94b0d1ee5/front/public/locales/en/errors.json)